runtime/gc_vlhgc/RegionBasedOverflowVLHGC.cpp

```diff
@@ -89,7 +89,7 @@ MM_RegionBasedOverflowVLHGC::tearDown(MM_EnvironmentBase *env)
  * Empty a packet on overflow
  * 
  * Empty a packet to resolve overflow by dirtying the appropriate 
- * cards for each object withing a given packet
+ * cards for each object with-in a given packet
```

Comment:
I think this should be `within`. I think there are other similar changes that should be adjusted likewise.